packages/smart-contracts/migrations/2_deploy_contracts.js

```diff
@@ -5,12 +5,16 @@ const EthereumProxy = artifacts.require('./EthereumProxy.sol');
 const ERC20FeeProxy = artifacts.require('./ERC20FeeProxy.sol');
 
 const erc20 = artifacts.require('./TestERC20.sol');
+const erc20Alpha = artifacts.require('./ERC20Alpha.sol');
```

Comment:
This is one of the only exceptions actually, we need to deploy a second ERC20 for integration tests from payment-processor for instance. But not from a separate contract, I fixed that already, thanks for your feedback.

NB: since we now have 2 ERC20 tokens, it would probably help debugging to name them differently.